depends/common/gli/gli.txt

```diff
@@ -0,0 +1 @@
+gli https://github.com/g-truc/gli/archive/dd17acf9cc7fc6e6abe9f9ec69949eeeee1ccd82.tar.gz
```

Comment:
Does the latest release not work?

Reply:
About `GLI` there is the last release (~2016) no more compatible with current GLM, that's why we come not around to use it, Also GLM noted it somewhere on his Website (not sure where, a bit ago where I read it)